Node Status Monitor/NodeJS/api/auth.js

```diff
@@ -0,0 +1,50 @@
+import fetch from 'node-fetch';
+import config from '../config/config.js';
+import log from '../utils/logger.js';
+
+let token_url = config.token_url;
+
+/*
+funtion to get New Token
+*/
+function get_token(clientId, clientSecret) {
+  return new Promise((resolve, reject) => {
+    log.info("-------------------- geting token  --------------------")
+    fetch(token_url,
+      {
+        method: 'POST',
+        body: 'grant_type=client_credentials&client_id=' + clientId + '&client_secret=' + clientSecret,
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
+      })
+      .then(res => res.json())
+      .then(json => {
```

Comment:
Can `json` be `null` or `undefined`? Need to protect against those exceptions.

Reply:
We are handling try catch in parent function 
monitor_node_status